include/subnet_cadmium.hpp

```diff
@@ -104,8 +105,8 @@ class Subnet {
             TIME next_internal;
             if (state.transmiting) {
                 std::initializer_list<int>
-				time = {0, 0, static_cast < int >
-				       (round(distribution(generator)))};
+	            time = {0, 0, static_cast < int >
```

Comment:
No space is required while type casting. 
i.e.., time = {0, 0, static_case<int>

Reply:
Resolved

Reply:
there will not be any space between static_cast and <int> as well.

Reply:
Resolved